initScripts/aarch32/Ubuntu_16.04/Docker_18.03.sh

```diff
@@ -37,7 +37,8 @@ export DEFAULT_TASK_CONTAINER_MOUNTS="-v $BUILD_DIR:$BUILD_DIR \
   -v $REQEXEC_DIR:/reqExec"
 export TASK_CONTAINER_COMMAND="/reqExec/$NODE_ARCHITECTURE/$NODE_OPERATING_SYSTEM/dist/main/main"
 export DEFAULT_TASK_CONTAINER_OPTIONS="-d --rm"
-
+export SHIPPABLE_DIND_IMAGE="docker:18.03.1-dind"
+export SHIPPABLE_DIND_CONTAINER_NAME="shippable-dind-conainer"
```

Comment:
Typo: `conainer` -> `container`. Seems redundant, though. Why not just call it `shippable-dind`?